cmd/license-lint/collector.go

```diff
@@ -160,14 +168,19 @@ var supportedLicenseFilenames = map[string]struct{}{
 }
 
 // find all license files in the given directory tree
-func findLicenseFiles(path string) ([]string, error) {
+func findLicenseFiles(basepath string) ([]string, error) {
 	var result []string
-	err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+	err := filepath.Walk(basepath, func(path string, info os.FileInfo, err error) error {
 		if info == nil {
 			return fmt.Errorf("unable to get information on %s: %v", path, err)
 		}
 
-		if !info.IsDir() {
+		if info.IsDir() {
+			if path == filepath.Join(basepath, "licenses") {
+				// don't recurse into this since it holds downstream licenses
```

Comment:
you mean upstream licenses here for the comment?

Reply:
Yes, fixed.